BeeTracker/app/src/main/java/com/blueprint/foe/beetracker/CaptureFragment.java

```diff
@@ -0,0 +1,51 @@
+package com.blueprint.foe.beetracker;
+
+import android.app.Fragment;
```

Comment:
Which android SDKs are you planning to support? If you're supporting older versions it's recommended to user support libraries because they have new features of android and support older versions.

https://stackoverflow.com/questions/18271429/difference-between-android-support-v7-appcompat-and-android-support-v4